modules/docker/stop_test.go

```diff
@@ -20,7 +20,7 @@ func TestStop(t *testing.T) {
 
 	// choosing a unique port since 80 may not fly well on test machines
 	port := "13030"
-	testURL := "http://localhost:" + port
+	testURL := "http://" + GetDockerHost() + ":" + port
```

Comment:
NIT: I know the code originally was written with concat, but it's more idiomatic in Go to use:

```suggestion
	testURL := fmt.Sprintf("http://%s:%d", GetDockerHost(), port)
```